pkg/apis/operator/v1alpha1/tektonconfig_types.go

```diff
@@ -95,13 +90,13 @@ type TektonConfigSpec struct {
 type TektonConfigStatus struct {
 	duckv1.Status `json:",inline"`
 
-	// The version of the installed release
+	// The profile installed
 	// +optional
-	Version string `json:"version,omitempty"`
+	Profile string `json:"profile,omitempty"`
 
-	// The url links of the manifests, separated by comma
+	// The version of the installed release
 	// +optional
-	Manifests []string `json:"manifests,omitempty"`
```

Comment:
Are we breaking the status API by removing this ?

Reply:
yes

Reply:
previously it was nil though.. we didn't use to add anything to that field in TektonConfig